Sky Atlas/Sky Atlas.html

```diff
@@ -2058,10 +2066,10 @@ <h1>Sections & Primary Docs</h1>
           <td>Emergency Response System - Emergency Response - Emergency Response Group Membership</td>
           <td>Active Data Controller</td>
           <td><p>The membership of the Emergency Response Group is defined as Active Data in <dfn>A.1.8 - Emergency Response Group Current Membership</dfn>.</p><p>The Active Data is updated as follows:</p>
-            <ul>
-              <li>The Responsible Party is the Support Facilitators.</li>
-              <li>The Update Process must follow the protocol for ‘Direct Edit’.</li>
-            </ul>
+          <ul>
+            <li>The Responsible Party is the Governance Facilitators.</li>
+            <li>The Update Process must follow the protocol for ‘Direct Edit’.</li>
+          </ul>
```

Comment:
add back indentations to remain consistent with the rest of the file

```suggestion
            <ul>
              <li>The Responsible Party is the Governance Facilitators.</li>
              <li>The Update Process must follow the protocol for ‘Direct Edit’.</li>
            </ul>
```